### PR TITLE
Fix openstack CI

### PR DESCRIPTION
**Pull request by tomassedovic**
#### What does this PR do?
This attempts to fix the end to end CI by adding manual checks to ensure the router and registry pods are actually up.

If this works, we may want to turn that into a post-install playbook.

#### How should this be manually tested?
No manual testing require, just verify the e2e ci passed.

#### Is there a relevant Issue open for this?
n/a

#### Who would you like to review this?
cc: @tzumainn @Tlacenka  @bogdando PTAL

### CODE REVIEW

**Comment by tomassedovic** [comment]
Yay, the CI passed. I'm going to do a very naughty thing and merge this myself, because this is blocking several open pull requests and there's no one who can approve this in my time + zone.

**Comment by tomassedovic** [comment]
Ha! Or not. Turns out I can't approve my patches. Well played, github, well played.

**Comment by tomassedovic** [comment]
@e-minguez @pschiffe any chance you can give this a look & approve? The openstack end to end CI is broken without it.

File: ci/openstack/install-openshift.sh [hunk 1]
```diff
@@ -18,3 +18,41 @@ export INVENTORY="$PWD/playbooks/provisioning/openstack/sample-inventory"
...
 
 ansible-playbook --become --timeout 180 --user openshift --private-key ~/.ssh/id_rsa -i "$INVENTORY" ../openshift-ansible/playbooks/byo/config.yml -e @extra-vars.yaml
+
+
+echo Waiting for the router to come up
+for i in $(seq 15); do
+    if ansible -i "$INVENTORY" masters --user openshift --private-key ~/.ssh/id_rsa -m shell -a 'oc get pod | grep router | grep Running'; then
+        echo Router is running
+        break
+    elif ansible -i "$INVENTORY" masters --user openshift --private-key ~/.ssh/id_rsa -m shell -a 'oc get pod | grep router | grep Failed'; then
```
Comment (e-minguez) [inline ci/openstack/install-openshift.sh]: You can avoid use shell ansible module and grep as:

```
oc get pods -l deploymentconfig=router -n default
```

Comment (e-minguez) [reply ci/openstack/install-openshift.sh]: Silly comment from my side, otherwise LGTM

Comment (tomassedovic) [reply ci/openstack/install-openshift.sh]: No, that's great. One question though, doesn't this just print out the status? We need the command to fail if it's not Running or Failed. The last grep fails there which gets propagated to Ansible.

Comment (e-minguez) [reply ci/openstack/install-openshift.sh]: Yep, you are right, grep is needed (I think jsonpath supports true or false output as well https://github.com/kubernetes/kubernetes/pull/39063 but I've been playing for 10 minutes and couldn't get it :D)

**Pull request merged**